### PR TITLE
SRVKS-509, remove operators.operatorframework.io/internal-objects ann…

### PR DESCRIPTION
**Pull request by maschmid**
…otation

reverts SRVKS-317 and SRVKE-207

### CODE REVIEW

File: olm-catalog/serverless-operator/1.6.0/serverless-operator.v1.6.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -68,7 +68,6 @@ metadata:
       Provides a collection of API's based on Knative to support deploying and serving
       of serverless applications and functions.
     repository: https://github.com/openshift-knative/serverless-operator
-    operators.operatorframework.io/internal-objects: '["knativeservings.operator.knative.dev"]'
```
Comment (bbrowning) [inline olm-catalog/serverless-operator/1.6.0/serverless-operator.v1.6.0.clusterserviceversion.yaml:71]: We shouldn't change the 1.6.0 CSV. If we want to fix this for 1.6.x, we'd need to create a new 1.6.1 release I believe.

Comment (maschmid) [reply olm-catalog/serverless-operator/1.6.0/serverless-operator.v1.6.0.clusterserviceversion.yaml]: right, thanks!

**Comment by bbrowning** [comment]
You'll still need approval from an OWNER, but lgtm.

/lgtm

**Comment by jcrossley3** [comment]
why aren't you an owner, @bbrowning 
/lgtm
/approve

**Pull request merged**